### PR TITLE
Main

### PR DESCRIPTION
**Pull request by martinboller**
## What

Additional and changed scripts for maintaining Greenbone and automating the creation of credentials, schedules, alerts, targets, tasks, tags and filters.

## Why

To automate the creation of a completely configured Greenbone Scanner

## References

Original gvm-tools scripts.

- [ ] Tests
[Install new scanner], alternatively
[clean-scanner]
[Prepare relevant CSV files for create-*.gmp.py scripts]
[create Credentials]
[Create Schedules
[Create Alerts]
[Create Targets]
[Create tasks}
[Create Tags]
[Create filters]
[List objects created w. list-*.gmp.py]
[Run task(s)]
[export-csv-report.gmp.py]
[export-pdf-report.gmp.py]
[Verify Reports]

### CODE REVIEW

File: scripts/create-alerts-from-csv.gmp.py [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+def main(gmp: Gmp, args: Namespace) -> None:
+    # pylint: disable=undefined-variable
+    if args.script:
+        args = args.script[1:]
+
+    parsed_args = parse_args(args=args)
+
+    print(
+        "Creating alerts.\n"
+    )
```
Comment (y0urself) [inline scripts/create-alerts-from-csv.gmp.py:242]:
```suggestion
    print("Creating alerts.\n")
```

File: scripts/create-alerts-from-csv.gmp.py [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
```
Comment (y0urself) [inline scripts/create-alerts-from-csv.gmp.py:21]:
```suggestion
# SPDX-FileCopyrightText: 2024 Martin Boller
#
# SPDX-License-Identifier: GPL-3.0-or-later
#

# Loosely based on Greenbone sample scripts
# As provided by Greenbone in the gvm-tools repo
```

**Review by y0urself — CHANGES_REQUESTED** [review]
Thank you for your contribution.
From what I see on the first view, you will probably need to run this with our CI tooling, because I expect these to fail (linting and formatting with `ruff` and `black`!).
I also noticed your copyright header, you may want to change these into our new format, that reduces the overhead in LoC for the header.

I will take another deep dive through your contribution tomorrow!

**Comment by y0urself** [comment]
@martinboller I would really love to merge this - is there any chance, that you fix up the stuff?

**Comment by martinboller** [comment]
Will these changes satisfy requirements or am I missing something (or everything)

**Comment by y0urself** [comment]
> Will these changes satisfy requirements or am I missing something (or everything)

No actually my suggestions were just examples, e.g. the Copyright header should have been updated in every file ...
Also running `black` and `ruff` in every file has not been done ...

**Comment by y0urself** [comment]
merged through #1112

**Pull request closed without merging**